mitremap-notebook/MitreMap - Infer MITRE technique from Threat Intel Data.ipynb

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
<p>This might be a dumb question but would this work if I was running Jupyter on Windows?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I tested this on my windows machine - Is that what you mean?